### PR TITLE
Update notes for Job Fair

### PR DESCRIPTION
**Pull request by mjang**
I don't know if I have correct sub-bullet syntax. And I don't know if I've PR'd the correct file.

But I think these notes will help for future virtual conferences.

### CODE REVIEW

**Comment by mjang** [comment]
I think the build error is not related to the PR:

Screenshot from 2021-04-30 06-23-09

File: docs/conf/prague/2021/job-fair.rst [hunk 1]
```diff
@@ -14,3 +14,26 @@ Schedule
...
+
+Tips for Attendees
+------------------
+
+If you're attending the Virtual Job Fair, make sure you're signed in to Hopin.
+
+* Select Expo in the left-hand pane.
+* Select the company of your choice.
+* You now have the following choices:
+  * Chat with attendees from the company under the Chat tab.
```
Comment (mjang) [inline docs/conf/prague/2021/job-fair.rst:28]: Hope this sets up sublist correctly:

```suggestion

  * Chat with attendees from the company under the Chat tab.
```

File: docs/conf/prague/2021/job-fair.rst [hunk 2]
```diff
@@ -14,3 +14,26 @@ Schedule
...
+  * Select the button to "Share Audio and Video". You can then chat "Face to Face" with company representatives.
+
+Tips for Sponsors
+-----------------
+
+If you want to maximize your contacts with potential new employees, make sure to:
+
+* Set up moderator privileges. To minimize delays, set this up **before the conference**. If needed, you can also request access from the designated Job Fair volunteer from the Write the Docs conference.
+* When you're a moderator, people who want to "Share Audio and Video" will appear under the "Moderator Panel" at the bottom of the screen.
```
Comment (mjang) [inline docs/conf/prague/2021/job-fair.rst]:
```suggestion
* When you're a moderator, people who want to "Share Audio and Video" will appear under the "Moderator Panel" at the bottom of the screen. When you select one or more persons, they'll join you in the video chat.
```

File: docs/conf/prague/2021/job-fair.rst [hunk 3]
```diff
@@ -14,3 +14,26 @@ Schedule
...
+
+  * Chat with attendees from the company under the Chat tab.
+  * Select the button to "Share Audio and Video". You can then chat "Face to Face" with company representatives.
+
+Tips for Sponsors
+-----------------
+
+If you want to maximize your contacts with potential new employees, make sure to:
+
+* Set up moderator privileges. To minimize delays, set this up **before the conference**. If needed, you can also request access from the designated Job Fair volunteer from the Write the Docs conference.
```
Comment (mxsasha) [inline docs/conf/prague/2021/job-fair.rst]: I don't think they'll have any idea what moderator privileges setup means. How I've been doing it behind the scenes is adding everyone that seems to work for that sponsor as a moderator.

```suggestion
* Make sure your booth staff is registered in the online platform before the conference starts. All ticket holders will receive details about this a few days before the conference. These staff will be set up as moderators for the booth If needed, you can also request additional moderators to be set up from the designated Job Fair volunteer from the Write the Docs conference.
```

File: docs/conf/prague/2021/job-fair.rst [hunk 4]
```diff
@@ -14,3 +14,26 @@ Schedule
...
+* When you're a moderator, people who want to "Share Audio and Video" will appear under the "Moderator Panel" at the bottom of the screen. When you select one or more persons, they'll join you in the video chat.
```
Comment (mxsasha) [inline docs/conf/prague/2021/job-fair.rst]:
```suggestion
* When you're a moderator, you can always share your audio and video. Other attendees who want to "Share Audio and Video" will appear under the "Moderator Panel" at the bottom of the screen. When you select one or more persons, they'll join you in the video chat.
```

**Review by mxsasha — COMMENTED** [review]
Suggested a few changes. Feel free to mash up my suggestions though :)

File: docs/conf/prague/2021/job-fair.rst (hunk 3 above)
Comment (mjang) [reply docs/conf/prague/2021/job-fair.rst]: I appreciate where you're going, I hope this makes your intent more clear, @mxsasha .

```suggestion
* Make sure your booth staff is registered in the online platform before the conference starts. All sponsor ticket holders will receive details about this a few days before the conference. These staff will be set up as moderators for your booth. As needed, you can also request additional moderators from the designated conference Job Fair volunteer.
* ```

File: docs/conf/prague/2021/job-fair.rst (hunk 4 above)
Comment (mjang) [reply docs/conf/prague/2021/job-fair.rst]: Yup, agreed.

File: docs/conf/prague/2021/job-fair.rst [hunk 5]
```diff
@@ -14,3 +14,27 @@ Schedule
...
+  * Chat with attendees from the company under the Chat tab.
+  * Select the button to "Share Audio and Video". You can then chat "Face to Face" with company representatives.
+
+Tips for Sponsors
+-----------------
+
+If you want to maximize your contacts with potential new employees, make sure to:
+
+* Make sure your booth staff is registered in the online platform before the conference starts. All sponsor ticket holders will receive details about this a few days before the conference. These staff will be set up as moderators for your booth. As needed, you can also request additional moderators from the designated conference Job Fair volunteer.
+* ```
```
Comment (mxsasha) [inline docs/conf/prague/2021/job-fair.rst]: Typo?
```suggestion
```

**Review by mxsasha — APPROVED** [review]
Looks great. I went ahead and committed Mike's suggestion for the second tip and removed that "*````" typo.

**Pull request merged**